dev/docker-compose.yaml

```diff
@@ -156,6 +156,7 @@ services:
 
       KEYCLOAK_CPRO_BACKCHANNEL_LOGOUT_URL: https://${BASE_DOMAIN}/auth/oidc/logoutCallback
       KEYCLOAK_SHL_CREATOR_BASE: https://shl-creator.${BASE_DOMAIN}
+      KEYCLOAK_SHL_CREATOR_LOGOUT_REDIRECT_URL: https://${BASE_DOMAIN}/users
```

Comment:
@daniellrgn 

this doesn't appear to be used by any templated placeholder in the keycloak JSON, am I missing something?